### PR TITLE
Fixes #3983 adds Greek language

### LINKED ISSUE #3983: Add Greek (el) language as supported language
## Description

Add [Greek](https://weblate.securedrop.org/projects/securedrop/securedrop/el/) language as a supported language.

## User Research Evidence

The language is fully translated and reviewed in the weblate.

### PR DESCRIPTION
**Pull request by kushaldas**
Greek is 100% translated and reviewed.

## Status

Ready for review

## Description of Changes

Fixes #3983

Changes proposed in this pull request:

## Testing

Follow the steps at https://docs.securedrop.org/en/release-0.10.0/development/i18n.html#merging-translations-back-to-develop and we should see files related to `el`. 

Next,

- [ ] Add Greek as a supported language via securedrop-admin sdconfig, the tester should verify site-specific contains the locale for Greek
- [ ] Verify that change can be applied to the server, i.e. securedrop-admin install updates config.py
- [ ] The menu item appears as expected in the source/journalist interfaces
- [ ] Follow steps below for page layout tests

These are new translations files (covered by a generic apparmor inclusion) and updates that can be verified if you speak the language by switching to the corresponding language using the menu. Ensuring there is no breakage because of placeholders incorrectly copied over can be done with:

```
bin/dev-shell bash
source bin/dev-deps
run_xvfb & run_redis & run_x11vnc & run_sass --watch & urandom & maybe_create_config_py
./i18n_tool.py translate-messages --compile
setting config.py line SUPPORTED_LOCALES = ['el','ar','de_DE','es_ES','fr_FR','nb_NO','nl','pt_BR','zh_Hant','en_US','it_IT','tr', 'hi', 'ru', 'sv']
running
$ PAGE_LAYOUT_LOCALES='el,ar,de_DE,es_ES,fr_FR,nb_NO,nl,pt_BR,zh_Hant,it_IT,tr,hi,ru,sv' pytest -v --page-layout tests/pageslayout
...
```

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

### If you made changes to the server application code:

- [ ] Linting (`make ci-lint`) and tests (`make -C securedrop test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

### If you made changes to documentation:

- [ ] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: securedrop/i18n_tool.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class I18NTool(object):
         'de_DE': {'name': 'German', 'desktop': 'de_DE', },
         'es_ES': {'name': 'Spanish', 'desktop': 'es_ES', },
         'fr_FR': {'name': 'French', 'desktop': 'fr', },
+        'el': {'name': 'French', 'desktop': 'el', },
```
Comment (redshiftzero) [inline securedrop/i18n_tool.py]: the `name` should be "Greek"

Comment (kushaldas) [reply securedrop/i18n_tool.py]: Yup, and eeks :(

**Comment by kushaldas** [comment]
>We should run the i10n tool which will update this file which should then be committed.

@redshiftzero do you want that `l10n.txt` file also to be included in this PR?

**Comment by redshiftzero** [comment]
> do you want that l10n.txt file also to be included in this PR?

Yes, unless there's a reason not to?

**Comment by kushaldas** [comment]
>Yes, unless there's a reason not to?

I was just asking :)

**Comment by kushaldas** [comment]
Pushed those two changes.

**Comment by redshiftzero** [comment]
The tests here helpfully inform us that we will need to also add the translation directory for Greek for this PR. Let's append the translation updates (not just for Greek but for all supported languages) to this PR when they are 100% translated/reviewed _or_ when it's approaching 10 AM Pacific time, whichever comes first. @kushaldas please run through the test plan at that time to ensure we did not forget anything that needs to be updated. 

Then others in the morning (Pacific time) shift should also run through the testing as described in this PR once those translation files are in.

**Comment by redshiftzero** [comment]
Just writing this out explicitly as there is some confusion about the purpose of the test plan and who should be testing what (perhaps we can discuss this at the engineering meeting on Thursday): for each PR, at least two people need to verify that changes do what is advertised. The submitter should test their own changes do what is advertised unless there is a compelling reason not to, i.e. "I had to sign off for the day and I did not get a chance to run through full testing here" is a fine reason, just note that on the PR so another person can pick up your work. We have a checkbox in the PR template in order to indicate whether or not this was done. 

In addition to the submitter, either the reviewer, or a QA participant should also test that changes do what is advertised (or that automated tests that have been added sufficiently test the change). If we think that the manual test plan is too onerous, we should discuss how to strike the right balance in the PR as part of review. The test plan is part of the PR template to clearly indicate what has been tested, which aids us both in collaboration and for the purpose of retrospectives.

**Comment by zenmonkeykstop** [comment]
On Tails 3.10.1, checked out `add_greek` branch and confirmed that:
- 'el' is listed as a language option during `sdconfig`
- if selected during `sdconfig`, 'el' is added as a supported language to site-specific YAML

**Comment by kushaldas** [comment]
In my local stage test, `Greek` is showing well, http://lnbnwqtzlo6uzei5.onion/?l=el

**Comment by redshiftzero** [comment]
So to be clear @kushaldas, this represents the outcome of your test?

- [x] Add Greek as a supported language via securedrop-admin sdconfig, the tester should verify site-specific contains the locale for Greek
- [ ] Verify that change can be applied to the server, i.e. securedrop-admin install updates config.py (**not tested**)
- [x] The menu item appears as expected in the source/journalist interfaces

**Comment by kushaldas** [comment]
>So to be clear @kushaldas, this represents the outcome of your test?

Correct.


- [x] Add Greek as a supported language via securedrop-admin sdconfig, the tester should verify site-specific contains the locale for Greek
- [ ] Verify that change can be applied to the server, i.e. securedrop-admin install updates config.py (**not tested**)
- [x] The menu item appears as expected in the source/journalist interfaces

**Pull request merged**